qualtran/resource_counting/_call_graph.py

```diff
@@ -94,6 +95,10 @@ def _generalize_callees(
     callee_counts: Dict[Bloq, Union[int, sympy.Expr]] = defaultdict(lambda: 0)
     if isinstance(raw_callee_counts, set):
         raw_callee_iterator: Iterable[BloqCountT] = raw_callee_counts
+        warnings.warn(
+            "build_call_graph returning sets is deprecated (got {raw_callee_counts})."
+            "Please change build_call_graph for this bloq to use a dictionary."
+        )
```

Comment:
add `DeprecationWarning` as the warning type?

Reply:
Done.